SmartDeviceLinkTests/DevAPISpecs/SDLStreamingVideoLifecycleManagerSpec.m

```diff
@@ -172,6 +177,10 @@ __block void (^sendNotificationForHMILevel)(SDLHMILevel hmiLevel, SDLVideoStream
 
                     someDisplayCapabilities.screenParams = someScreenParams;
 
+                    SDLVersion *oldVersion = [SDLVersion versionWithMajor:6 minor:0 patch:0];
+                    id globalMock = OCMPartialMock([SDLGlobals sharedGlobals]);
+                    OCMStub([globalMock rpcVersion]).andReturn(oldVersion);
```

Comment:
What is this for?

Reply:
so i can test that if the SDLVersion is 4.5+ it will run the code in the 'if' in `SDLStreamingVideoLifecycleManager.m`